climada/hazard/centroids/centr.py

```diff
@@ -538,8 +538,7 @@ def append(self, centr):
                 setattr(self, var_name, np.append(var_val, centr_val).
                         astype(var_val.dtype, copy=False))
 
-    @staticmethod
-    def join_centroid(centroids_list):
+    def union(self, *centroids_list):
```

Comment:
what about calling it `union_centroids`?
also you need to change line 1540 of hazard/base.py where it's called with its old name

Reply:
no wait i'm happy with just `union`.
i really need to think twice before expressing opinions today.

Reply:
:D and I have to see not to push commits that are unfinished. Thanks for the line 1540 reminder.